service/src/main/java/uk/gov/hmcts/reform/rd/client/OrganisationApi.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.rd.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestParam;
+import uk.gov.hmcts.reform.fpl.config.FeignConfiguration;
+import uk.gov.hmcts.reform.rd.model.Status;
+import uk.gov.hmcts.reform.rd.model.User;
+
+import java.util.List;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi.SERVICE_AUTHORIZATION;
+
+@FeignClient(name = "rd-professional-api", url = "${rd_professional.api.url}", configuration = FeignConfiguration.class)
```

Comment:
If not tight deadline I would have asked you to move it to separate library so that everyone can reuse it.

Reply:
I agree, personally I even think the owning teams could provide the clients, but definitely it should be shared.